src/AuthModule/Components/ResetPassword/ResetPassword.tsx

```diff
@@ -1,7 +1,226 @@
-import React from 'react'
+import React, { useState } from "react";
+import { Link, useNavigate } from "react-router-dom";
+import { useForm } from "react-hook-form";
+import Visibility from '@mui/icons-material/Visibility';
+import VisibilityOff from '@mui/icons-material/VisibilityOff';
+import {
+  TextField,
+  Button,
+  Typography,
+  Box,
+  Grid,
+  Container,
+  OutlinedInput,
+  InputAdornment,
+  IconButton,
+} from "@mui/material";
+import axios from "axios";
+import forgotImage from "../../../assets/resetPAss.png";
+import { baseUrl } from "../../../Constants/Components/Urls";
+import { toast } from "react-toastify";
+import styles from "./ResetPassword.module.css";
+import { OTPValidation, emailValidation, passwordValidation } from "../Validators/Validators";
+import { FormResetData } from "../../../interfaces/interfaces";
+
 
 export default function ResetPassword() {
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+  const navigate = useNavigate();
+
+  const [showPassword, setShowPassword] = useState<boolean>(false);
+  const [showConfirmPassword, setShowConfirmPassword] = useState<boolean>(false);
+
+  const handleClickShowPassword = () => setShowPassword((show) => !show);
+  const handleClickShowConfirmPassword = () => setShowConfirmPassword((show) => !show);
+
+  const { register, handleSubmit, formState: { errors }, watch } = useForm<FormResetData>();
+
+  const confirmPassword = watch("password");
+
+  async function callForgotPassApi(data: FormResetData) {
+    try {
+      setIsLoading(true);
+      const response = await axios.post(
+        `${baseUrl}/admin/users/reset-password`,
+        data
+      );
+      toast.success(response.data.message);
+      navigate("/Authentication");
+    } catch (error) {
+      console.error(error);
+      toast.error(error?.response?.data?.message || "There's a mistake.");
+    } finally {
+      setIsLoading(false);
+    }
+  }
+
+  const onSubmit = (data: FormResetData) => {
+    callForgotPassApi(data);
+  };
+
   return (
-    <div>ResetPassword</div>
-  )
+    <Container
+      maxWidth={"xl"}
+      sx={{
+        marginY: 0,
+        paddingY: 3,
+        display: "flex",
+        justifyContent: "center",
+        maxHeight: "100%",
+        minHeight: "100vh",
+      }}
+    >
+      <Grid
+        container
+        spacing={0}
+        sx={{
+          justifyContent: "space-around",
+        }}
+      >
+        <Grid item xs={12} sm={8} md={5}>
+          <Link to={"/"} className="logo"  style={{ "color" : "#3252df" ,"fontSize":"1.5rem" ,"textDecoration":"none" ,"fontWeight":600}}>
```

Comment:
It is highly reccomended to stick with the MUI theme's colors, font-sizes and weights. This will make it easier to update all the values from one place, and if we need to a feature like dark/light mode will be much easier to control from the MUI theme.